### ISSUE TITLE
spaced-comment implicitly adds space to marker

### ISSUE THREAD

**Issue by Alexsey** [eslint/eslint#9883]
**Tell us about your environment**

* **ESLint Version:** 4.16.0
* **Node Version:** 9.3.0
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:** 

<details>
<summary>Configuration</summary>

```yaml
---
parserOptions:
  ecmaVersion: 2017
  ecmaFeatures:
    experimentalObjectRestSpread: true
  sourceType: script
env:
  node: true
rules:
  spaced-comment:
    - error
    - always
    - markers:
      - global
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
/*globalABC*/
```
**What did you expect to happen?**
No error should be reported

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint: Expected space or tab after /*global in comment. (spaced-comment)

I have take this example with `global` from documentation. If you add space after `global` than no error would be risen

So it looks like `marker` is appended with space and there is no way to configure it

Why is it useful? Because sometimes to comment about next section of code something like
`/*********** Next Section **************/`
could be used. And there is no way configure the rule to allow it while having second parameter as `always`

**Comment by platinumazure** [comment]
I'm not sure I understand the use case.

Are you saying you want to enforce spacing in comments by default, but that anything starting with `global` (including more text) should count as a marker and should not need a space?

I'm being thrown off by "global" in particular since [`/* global */` comments have a special meaning in ESLint](https://eslint.org/docs/user-guide/configuring#specifying-globals), so I guess my question is, are you saying you need a string prefix to be supported as a marker, and not require a space immediately afterwards?

Have you tried using the `exceptions` [option](https://eslint.org/docs/rules/spaced-comment#options)? Will that do what you need?

**Comment by Alexsey** [comment]
Yes, `global` may be not the best choice to illustrate the real world use case, but you have understand everything correctly. And yes, I was trying to use `exceptions`, but it requires all string to be like provided pattern. So if in initial example replace `markers` with `exceptions` than only comments like `/*global*/` or `/*globalglobal*/` and like so would be acceptable

**Comment by kaicataldo** [comment]
Sorry for the delay here. It sounds like you're asking for an enhancement to this rule. If that's the case, would you mind following the instructions [here](https://eslint.org/docs/developer-guide/contributing/rule-changes) for rule changes?

**Comment by Alexsey** [comment]
Now I'm sorry for the late answer

Actually, the current behavior may be a bug - look at the last but one example in the [docs of the rule](https://eslint.org/docs/rules/spaced-comment) where the section is named `Examples of correct code for this rule with the "always" option combined with "markers"`:
```js
/*eslint spaced-comment: ["error", "never", { "markers": ["!<"] }]*/

//!<This is a line comment with a marker

/*!<this is a block comment with a marker
subsequent lines are ignored
*/
```
It doesn't have a space after marker! And if you change `"never"` to `"always"` (because section is about `"always"` and because with `"never"`, `"markers"` property doesn't change anything) it would become an error. Just like in my initial example

**Comment by nzakas** [comment]
Unfortunately, it looks like there wasn't enough interest from the team or community to implement this change. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to be implemented after 90 days tend to never be implemented, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue). This doesn't mean the idea isn't interesting or useful, just that it's not something the team can commit to.
